service/src/main/java/bio/terra/pipelines/stairway/imputation/AddWdsRowStep.java

```diff
@@ -0,0 +1,88 @@
+package bio.terra.pipelines.stairway.imputation;
+
+import bio.terra.pipelines.common.utils.FlightUtils;
+import bio.terra.pipelines.common.utils.PipelinesEnum;
+import bio.terra.pipelines.dependencies.common.HealthCheckWorkspaceApps;
+import bio.terra.pipelines.dependencies.sam.SamService;
+import bio.terra.pipelines.dependencies.stairway.JobMapKeys;
+import bio.terra.pipelines.dependencies.wds.WdsService;
+import bio.terra.pipelines.dependencies.wds.WdsServiceApiException;
+import bio.terra.pipelines.dependencies.wds.WdsServiceException;
+import bio.terra.stairway.*;
+import bio.terra.stairway.exception.RetryException;
+import org.databiosphere.workspacedata.client.ApiException;
+import org.databiosphere.workspacedata.model.RecordAttributes;
+import org.databiosphere.workspacedata.model.RecordRequest;
+
+/**
+ * This step creates or replaces a row to a WDS table specific to the pipeline that was launched
+ * currently it writes the flight id as the primary key and a hardcoded "scatter" value that will be
+ * replaced once inputs are being passed in from the user.
+ *
+ * <p>The step expects a cbas uri to be passed in through the working map
+ */
+public class AddWdsRowStep implements Step {
```

Comment:
reminder to update the createJob steps sheet with the step name you used and changes to working map updates (e.g. we don't need to store the wds row id because we're using the flight id)